content/doc/book/managing/nodes.adoc

```diff
@@ -14,9 +13,64 @@ endif::[]
 
 = Managing Nodes
 
-### Monitor and Restart Offline Agents
-This script can monitor and restart offline nodes if they are not disconnected manually.
+== Launch inbound agent via Windows Scheduler
+
+If you are having trouble getting the inbound agent installed as a Windows service (i.e., you followed https://wiki.jenkins.io/display/JENKINS/Installing+Jenkins+as+a+Windows+service[the instructions on installing the agent as a service here] but it didn't work), an alternative method of starting the service automatically when Windows starts is to use the Windows Scheduler. 
+
+We take advantage of the Windows Scheduler's ability to run command at system startup
 
+. Configure your node to use the "Launch agents by connecting it to the master" launch method
+* Click Save
+. Note the command required to launch the agent
+* On the new agent node's Jenkins page, note the agent command line shown. 
+** It will be like: java -jar agent.jar -jnlpUrl http://<JenkinsHostName>:8080/computer/<nodeName>/slave-agent.jnlp -secret <some_long_hex_string>
+. Obtain the agent.jar file and copy it to your new Windows agent node
+* In the command line noted in the last step, the "agent.jar" is a hyperlink. Click it to download the agent.jar file.
+* Copy the agent.jar file to a permanent location on your agent machine
+. Ensure that you have a java version available on your agent machine
+* If not, obtain and install a link:/doc/administration/requirements/java/[supported version] of Java
+. Run the command manually from a CMD window on your agent to confirm that it works
+* Open the CMD window
+* Run the command (the one like java -jar agent.jar -jnlpUrl http://<JenkinsHostName>:8080/computer/<nodeName>/slave-agent.jnlp -secret <some_long_hex_string>)
```

Comment:
Needs to be separated into a code fragment block for readability.

Needs the URL escaped so that it is not clickable on the page

Reply:
Is there a markdown hack to wrap code blocks because long code block breaks responsive page.
![image](https://user-images.githubusercontent.com/5441467/83396194-0b5af400-a415-11ea-8dc0-e4f96414ffdf.png)

Reply:
code blocks formatted in the latest merge request.